server/controllers/v1/tests/post.test.ts

```diff
@@ -71,6 +74,86 @@ describe('Test v1 - Post Controller', () => {
       expect(result.status).toBe(200);
     });
 
+    it('should allow creating new tags when making a post', async () => {
+      const author = await makeValidUser();
+      const tags = ['csc108', 'bananaPepper'];
+
+      const result = await postController.createPost(
+        author.id,
+        'This is a new post!',
+        'This is a new post!This is a new post!',
+        'location',
+        10,
+        tags
+      );
+
+      if (result.data.result) {
+        const postTags = await getPostTags(result.data.result.id);
+        expect(postTags?.map((t) => t.text).sort()).toEqual(tags.sort());
+      } else {
+        expect(true).toBe(false);
```

Comment:
if statements in tests aren't good practice. You should expect that `result.data.result` is not empty/null in place of the `if(...)`

Reply:
I know but I was getting TS errors if I didnt wrap it in this if statement

Reply:
On second thought, I'll update it and just do @ts-ignore

Reply:
Or if you know for sure that you'll get a result you can use `!`